app/consumers/recommendation_consumer.rb

```diff
@@ -1,10 +1,10 @@
 class RecommendationConsumer
   include Hutch::Consumer
 
-  consume "copro.recommendations.*"
+  consume "copro.recommendations.*.*"
```

Comment:
It's misleading. It doesnt work like a real regexp. `copro.recommendations.*` matches `copro.recommendations.lol`, but not `copro.recommendations.lol.kikou`.